### ISSUE TITLE
PNG images not rendering properly on .Net 6

### ISSUE THREAD

**Issue by leo2d** [ststeiger/PdfSharpCore#191]
### TL;DR Updating ImageSharp to v.1.0.4 solves the issue

I'm facing an issue due to the version of the ImageSharp used in the PdfSharpCore.csproj

This is an ImageSharp issue where the PNG is not completely rendered. If you look at the ImageSharp repo you'll see [others relating the same issue](https://github.com/SixLabors/ImageSharp/issues/1728) 

Fortunately it has already been fixed on the release [v1.0.4](https://github.com/SixLabors/ImageSharp/releases/tag/v1.0.4)

I did some tests here and its working properly after updating the ImageSharp version used in the PdfSharpCore.csproj. Also all the unit tests are still passing.

**How to reproduce:**

- Add _net6.0_ as a TargetFramework in the PdfSharpCore.csproj and PdfSharpCore.Test.csproj
- Go to the _CreateTestPdfWithImage_ test  and add some logic to save the file in a place you can open it (suggested code bellow)
```c#

            //ReadStreamAndVerifyPdfHeaderSignature(stream);

            var file = File.Create(Path.Combine(@"your_path_here","test_.pdf"));

            stream.Seek(0, SeekOrigin.Begin);

            stream.CopyTo(file);

            file.Close();
            file.Dispose();

```
 - Go and check the file to see the issue with the image(as described bellow)

.Net 6(ImageSharp 1.0.0):
![image](https://user-images.githubusercontent.com/28194654/141217038-2e249de4-8535-46ef-a4db-1e1d3dc9c275.png)

.Net 5/.Net 6(ImageSharp 1.0.4):
![image](https://user-images.githubusercontent.com/28194654/141216959-ff0bf112-2421-4f86-947d-137909def76c.png)
